### PR TITLE
display unnamed domains when host.domain-list is executed

### PR DESCRIPTION
**Pull request by Wescoeur**

### CODE REVIEW

File: rust/daemon/main.rs [hunk 1]
```diff
@@ -40,13 +40,15 @@ fn main () {
...
-
-      Ok(domains) => Ok(Value::from_iter(domains.into_iter().filter_map(|dom_info| {
-        let name = vm::get_name(&xs, dom_info.domain.into()).ok()?;
-        Some(json!({
+      Ok(domains) => Ok(Value::from_iter(domains.into_iter().map(|dom_info| {
+        let name = match vm::get_name(&xs, dom_info.domain.into()) {
+          Ok(vm_name) => vm_name,
+          Err(_) => String::from("(null)")
+        };
+        json!({
```
Comment (benjamreis) [inline rust/daemon/main.rs:48]: Why remothe the `Some`?

Comment (Wescoeur) [reply rust/daemon/main.rs:48]: Because we use `map` instead of `filter_map`. We don't remove values now.

Comment (benjamreis) [reply rust/daemon/main.rs:48]: :+1:

**Review by benjamreis — APPROVED** [review]

**Comment by benjamreis** [comment]
Okay to merge?

**Pull request merged**